docs/src/components/Prose.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+const Prose = styled.div`
+  font-family: ${({ theme }) => theme.orbit.fontFamily};
+  font-size: ${({ theme }) => theme.orbit.fontSizeTextNormal};
+  > * + * {
+    margin-top: ${({ theme }) => theme.orbit.spaceSmall};
+  }
+  h1 {
```

Comment:
What's the benefit of handling heading spacing and such here and not in the `mdx-components`? We have `spaceAfter` still, no?

I'm not certain the value of the basic font CSS in this component as opposed to more generally.

Reply:
My understanding is that the `spaceAfter` prop isn't necessarily going to be around forever. The only use case for it that I can think of is writing documents, which component libraries aren't really good at, they are intended for applications. We'll most probably want to add `margin-top` to headings, which is something that Orbit doesn't (and shouldn't) support, so I created this component to define in one place styles that define relations between elements in a document.

> I'm not certain the value of the basic font CSS in this component as opposed to more generally.

If by "more generally" you mean that I could apply `font-family` and `font-size` to the the entire HTML document, that's probably a better way to go, but I'll see how this goes.